### PR TITLE
[ui][fix] Fixes for database tests

### PR DESCRIPTION
**Pull request by mcada**
//test: @integrations-db-to-db-oracle12

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: utilities/src/main/java/io/syndesis/qe/bdd/validation/OpenshiftValidationSteps.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        KafkaTemplate.deploy();
+    }
+
+    @Given("^deploy HTTP endpoints")
+    public void deployHTTPEndpoints() {
+        HTTPEndpointsTemplate.deploy();
+    }
+
+    @And("^wait until \"([^\"]*)\" pod is reloaded$")
+    public void waitUntilPodIsReady(String podName) {
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/bdd/validation/OpenshiftValidationSteps.java]: reloaded or ready?

File: utilities/src/main/java/io/syndesis/qe/templates/MysqlTemplate.java [hunk 2]
```diff
@@ -106,10 +107,24 @@ public static void deploy() {
...
-        TestUtils.sleepIgnoreInterrupt(5000);
+        try {
+            OpenShiftUtils.getInstance().getDeploymentConfigs().stream().filter(dc -> dc.getMetadata().getName().equals(APP_NAME)).findFirst()
+                    .ifPresent(dc -> OpenShiftUtils.getInstance().deleteDeploymentConfig(dc, true));
+            OpenShiftUtils.getInstance().getServices().stream().filter(service -> APP_NAME.equals(service.getMetadata().getName())).findFirst()
+                    .ifPresent(service -> OpenShiftUtils.getInstance().deleteService(service));
+            TestUtils.sleepIgnoreInterrupt(5000);
+        } catch (Exception e) {
+            log.error("Error thrown while trying to delete mysql database. It is just deletion, it should not affect following tests.");
+            log.error(e.toString());
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/templates/MysqlTemplate.java]: log.error("Error thrown while trying to delete mysql database. It is just deletion, it should not affect following tests.", e)?

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**